conda-forge.yml

```diff
@@ -4,3 +4,6 @@ github:
 conda_build:
   error_overlinking: true
 conda_forge_output_validation: true
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
```

Comment:
@lgray how interested are you in the `ppc64le` builds? They haven't started yet (on Travis) after waiting for 40 minutes, so if they pass they will make builds very time consuming. If they are important we can try to keep them in, but I'm just wondering if this is a "nice to have" or "actually would be very important" platform.